AMTGBot/Bot.cs

```diff
@@ -25,7 +32,10 @@ public async void SendSingleInlineQueryAnswer(string queryId, InlineQueryResultB
             {
                 await BotClient.AnswerInlineQueryAsync(queryId, new[] { baseResult });
             }
-            catch { }
+            catch (Exception e)
```

Comment:
Meh, we shouldn't catch any `Exception`. What if it is, I don't know, BAD ALLOC exception? Or `Universe collapsed panic exception`? Instead, there should be caught only those exceptions thrown by the method. If something very bad happened, it's better to crash the bot (with catching Exception, logging, and rethrowing), than to keep working

Reply:
Okay, now catching bad request exception (when query is too old to answer).